### PR TITLE
Add coq-coqeal.2.0.3

### PR DESCRIPTION
**Pull request by proux01**
Add coq-coqeal.2.0.3

### CODE REVIEW

File: released/packages/coq-coqeal/coq-coqeal.2.0.3/opam [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  "Érik Martin-Dorel"
+  "Anders Mörtberg"
+  "Damien Rouhling"
+  "Pierre Roux"
+  "Vincent Siles"
+]
+
+url {
+  src: "https://github.com/coq-community/coqeal/archive/refs/tags/2.0.2.tar.gz"
+  checksum: "sha512=a1bd42c9d359a02221f211c8e748510ec7700dbd1dce080bcdfd0bb7e141c597c75d95a352e833ab74217e5067b51684be1a1629f3fdead531695025dba912a1"
```
Comment (palmskog) [inline released/packages/coq-coqeal/coq-coqeal.2.0.3/opam:54]:
```suggestion
  src: "https://github.com/coq-community/coqeal/archive/refs/tags/2.0.3.tar.gz"
  checksum: "sha512=4e11d64d15402a4b2b3708aa98bed88ef9a8221f4f2356493ac3d835b267b9b526878b0d4edc69eeb0f9ffdb9f4e8db3d5e867d1a15ef70d8c4f2ce3a723a419"
```

Comment (proux01) [reply released/packages/coq-coqeal/coq-coqeal.2.0.3/opam]: Damn, always forget that one

**Comment by palmskog** [comment]
@proux01 looks like CI passed just fine, just the "any" job that did an opam solver timeout. Is there any reason not to go out of draft and merge?

**Comment by proux01** [comment]
You're right, thanks, let's merge

**Pull request merged**